mainframer.sh

```diff
@@ -70,6 +70,22 @@ if [ -z "$REMOTE_COMMAND" ]; then
 	exit 1
 fi
 
+function formatTime {
+    local time=$1
+
+    local hours=$((time/3600))
+    local minutes=$(((time % 3600) / 60))
+    local seconds=$((time % 60))
+
+    if [ "$hours" -eq "1" ]; then HOURS_LABEL="hour"; else HOURS_LABEL="hours"; fi
+    if [ "$minutes" -eq "1" ]; then MINUTES_LABEL="minute"; else MINUTES_LABEL="minutes"; fi
+    if [ "$seconds" -eq "1" ]; then SECONDS_LABEL="second"; else SECONDS_LABEL="seconds"; fi
+
+    (( hours > 0 )) && printf "%d $HOURS_LABEL " ${hours}
+    (( minutes > 0 )) && printf "%d $MINUTES_LABEL " ${minutes}
+    (( seconds > 0 )) && printf "%d $SECONDS_LABEL \n" ${seconds}
```

Comment:
I think you can not really test this since the time it needs to sync is variable, I tested some different locally with putting sleep. Like `sh mainframer.sh sleep 10` maybe a sleep 0 would give you this behaviour?